qemu/seabios_iommu.py

```diff
@@ -0,0 +1,111 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_test
+from virttest import utils_disk
+from provider.storage_benchmark import generate_instance
+
+
+def check_data_disks(test, params, env, vm, session):
+    """
+    Check guest data disks (except image1)
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    :param vm: VM object
+    :param session: VM session
+    """
+    image_list = params.objects("images")
+    del image_list[0]
+    image_num = len(image_list)
+
+    error_context.context("Check data disks in monitor!", logging.info)
+    monitor_info_block = vm.monitor.info_block(False)
+    blocks = ','.join(monitor_info_block.keys())
+    for image in image_list:
+        if image not in blocks:
+            test.fail("drive_%s is missed: %s!" % (image, blocks))
+
+    error_context.context("Read and write on data disks!", logging.info)
+    os_type = params["os_type"]
+    if os_type == "linux":
+        sub_test_type = params.get("sub_test_type", "dd_test")
+        for image in image_list:
+            params["dd_if"] = "ZERO"
+            params["dd_of"] = image
+            utils_test.run_virt_sub_test(test, params, env, sub_test_type)
```

Comment:
I believe calling subtest is deprecated.

Reply:
Updated, using iozone instead. Could you please help review again? @lolyu Thanks!